includes/model/class-blog.php

```diff
@@ -368,4 +377,36 @@ public function get_indexable() {
 			return false;
 		}
 	}
+
+	/**
+	 * Update User profile attributes
+	 *
+	 * @param string $key The attribute to update.
+	 * @param mixed $value The new value. Possible values:
+	 *                     - name: The User-Name.
+	 *                     - summary: The User-Description.
+	 *                     - icon: The User-Icon.
+	 *                     - header: The User-Header-Image.
+	 * @return bool True if the attribute was updated, false otherwise.
+	 */
+	public function save( $key, $value ) {
+		switch ( $key ) {
+			case 'name':
+				return \update_option( 'blogname', $value );
```

Comment:
That seems a bit dangerous, you might not realize that you're changing the whole blog name. Should this just be an override?

Reply:
It's tricky. Once we introduce an override, now you have to go into wp-admin to also change the blogname, if that's what you want to do, and/or being able to delete the override, falling back to the blogname, which you can now only edit in wp-admin. If you want a consistent name, you have to go to wp-admin. I'm thinking more of somebody who never wants to go there at all.

I'm erring on the side of consistency, not on the side of safety. And if somebody doesn't like the change they made, they can go back and change it again, without having to go to wp-admin.

At least that's the kind of thinking I was doing when I made it behave this way